src/app/pages/courses/components/add-course/add-course.component.html

```diff
@@ -0,0 +1,8 @@
+<section class="add-course-container">
```

Comment:
according to BEM block and element separates with double __ or double --, the reason is not to read as single class. But maybe it is not according to BEM.

Reply:
Yep, I'm not sure that BEM is relevant with components approach. In my opinion there is no reason to add an extra level of abstraction.